in_progress/roles.md

```diff
@@ -0,0 +1,109 @@
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise
+
+## Core team
+
+The core team are community members that have demonstrated continued commitment to the project through ongoing contributions.
+They have shown they can be trusted to maintain the scverse with care.
+Team members appear as organization members on the scverse [GitHub organization](https://github.com/orgs/scverse/people) and are on our [@scverse/core](https://github.com/orgs/scverse/teams/core) GitHub team (**TODO**).
+
+There are sub-roles within the core team depending on the kinds covering different kinds of contributions and responsibilities.
+
+### Core developers
+
+Core developers are members of the core team who have made signifigant code contributions to the `scverse`.
+Becoming a core developer allows contributors to merge approved pull requests, cast votes for and against merging a pull-request, and be involved in deciding major changes to the API, and thereby more easily carry on with their project related activities. 
+Core developers are on our [@scverse/core-devs](https://github.com/orgs/scverse/teams/core-devs) GitHub team (**TODO**).
+Core developers are expected to review code contributions while adhering to the [core developer guide](CORE_DEV_GUIDE.md) (**TODO**).
+
+All core packages must have at least one core developer as a maintainer.
+New core developers can be nominated by any existing core developer. 
+While there is no hard-and-fast rule about who can be nominated, ideally, they should have: 
+
+* Been part of the community for at least two months
+* Contributed significant changes of their own
+* Contributed to the discussion and review of others' work
+* Collaborated in a way befitting our community values
+
+After nomination voting will happen on a private mailing list by the core team.
+While it is expected that most votes will be unanimous, a two-thirds majority of the cast votes is enough.
+
+Primary maintainers of new core packages, who are not already core developers, will be invited to join.
+
+## Steering Council
+
+The Steering Council (SC) members are core developers who have additional responsibilities to ensure the smooth running of the project.
+SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to the project itself.
```

Comment:
this language implies the PIs are on the steering council

Reply:
to clarify, we should clarify this is regarding funding granted to the org versus to a project (potentially through a lab)

Reply:
yes